source/retryable-writes/retryable-writes.rst

```diff
@@ -514,13 +518,16 @@ Consider the following pseudo-code:
       if ( ! isRetryableWritesSupported(server)) {
         throw originalError;
       }
-    }    
+    }
   }
 
 
 ``handleError`` in the above pseudocode refers to the function defined in the
 `Error handling pseudocode`_ section of the SDAM specification.
 
+Handshake errors which are considered retryable will be handled by the SDAM logic causing an eventual
+server selection timeout. TODO DO NOT MERGE!!!
```

Comment:
Is this referencing a comment from @ShaneHarvey's previous review?

Reply:
Yes I meant to leave a comment on here before logging off yesterday, in the CSOT channel there was discussion of reverting the new retryable changes. So this sentence is in place of my pseudo code changes which got removed with the merge. This is allegedly how handshake errors would be retried with the new paradigm, SDAM would be the exit condition to the indefinite loop, but the test scenarios we've come up with here, are specifically demonstrating the scenario where hellos for the monitor are succeeding, but the handshake for operations aren't. So either handshakes have to be special case like the socketTimeout errors are, or we reconsider how the loop is managed.

Reply:
I've updated the transaction tests now, Shane cleverly suggested using multiple sessions to avoid killing the transaction before we're able to run abort or commit. Now the dropCollections trick should empty the pool right before those two final operations.

Reply:
This is removed now, no longer relevant due to the revert.